Marlin/src/lcd/extui/lib/dgus_creality/DGUSScreenHandler.h

```diff
@@ -227,11 +227,12 @@ class DGUSScreenHandler {
   }
 
   // Send an icon to the display, depending on whether it is true or false
-  template<unsigned int value_if_true, unsigned int value_if_false>
+  template<unsigned int value_if_true, unsigned int value_if_false, unsigned int value_offset>
```

Comment:
The point of the `value_if_true` and `value_if_false` parameter is to give the icon ID for the icons in question. So for sound etc you'd create a new `ICON_SOUND_ON` and `ICON_SOUND_OFF` constants, and then reference that 😉

Reply:
Code updated